tensorflow-framework/src/main/java/org/tensorflow/framework/metrics/impl/ConfusionMatrixEnum.java

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=======================================================================*/
+package org.tensorflow.framework.metrics.impl;
+
+/** Enumerate the values for a confusion matrix. */
+public enum ConfusionMatrixEnum {
+  /** These are cases in which the prediction is true, and reality is true. */
+  TRUE_POSITIVES("tp"),
+  /** These are cases in which the prediction is false, and reality is true. */
```

Comment:
The comments don't match the enums.

Reply:
Hows this?

```
  /** These are cases in which the prediction is true, and reality is true. */
  TRUE_POSITIVES("tp"),
  /** These are cases in which the prediction is true, and reality is false. */
  FALSE_POSITIVES("fp"),
  /** These are cases in which the prediction is false, and reality is true. */
  TRUE_NEGATIVES("tn"),
  /** These are cases in which the prediction is false, and reality is false. */
  FALSE_NEGATIVES("fn");
```

Reply:
`TRUE_NEGATIVES` would be both prediction and reality false.

Reply:
That is why they call it Confusion Matrix :-)
How's this:

```
 /** These are cases in which the prediction is true, and reality is true. */
  TRUE_POSITIVES("tp"),
  /** These are cases in which the prediction is true, and reality is false. */
  FALSE_POSITIVES("fp"),
  /** These are cases in which the prediction is false, and reality is false. */
  TRUE_NEGATIVES("tn"),
  /** These are cases in which the prediction is false, and reality is true. */
  FALSE_NEGATIVES("fn");
```

Reply:
:-) Sold!